pkg/applicationmanager/controllers/applicationclone.go

```diff
@@ -545,7 +546,7 @@ func (a *ApplicationCloneController) prepareResources(
 			pvNameMappings,
 			clone.Spec.IncludeOptionalResourceTypes,
 			nil,
-			nil,
+			&opts,
```

Comment:
Can we fix it in the prepareResources function itself?
All the callers need not be modified because of this.